articles/mobile-automation-testing-with-appium/index.md

```diff
@@ -0,0 +1,99 @@
+Appium is an automation testing tool that can be used for validating mobile browsers and mobile applications. This tool is widely used in mobile automation testing because it is free and can support both iOS and Android platforms. Mobile apps have become an important element in our lives because they provide a smart way of doing things (eg. shopping, paying bills, chatting, and booking flights) using mobile devices.
+
+Many mobile apps are constantly being developed to solve various real-life problems. Before these apps are commissioned for use, they are tested to validate their usability, quality, and functionality.
+
+This article takes readers through Appium mobile automation testing tool and discusses its key features. It also provides an overview of Appium architecture and explains how it works.
+
+### Introduction to Appium
+Appium is a popular mobile automation testing tool used for testing mobile browsers and mobile applications. This software has been limited to mobile application testing, with a specific focus on iOS, Firefox OS, and Android applications. Various programming languages are supported in Appium automation testing. These include Python, PHP, Java, and Perl.
+
+Appium provides tools for the automation testing of real physical objects, emulators, and simulators. Automation testing in Appium does not depend on the operating system of a mobile device. This is because its framework can translate the driver commands into Android or iOS commands that do not depend on the OS type. Instead, these commands depend on the type of mobile device.
+
+The following are some of the main types of mobile apps that can be tested using Appium automation testing:
+
+- **Native apps:** These are platform-specific apps that are developed using Android, iOS, or Windows SDKs. They operate on specific platforms. For example, either iOS or Android. These apps do not have features for web browser navigation.
+- **Hybrid apps:** These are apps that can work well on a certain platform and still have features for web browser navigation. They are a combination of mobile applications and features for web browser navigation.
+- **Web apps:** These are mobile apps that can be accessed using browsers of mobile devices (built-in).

```

Comment:
I have used web browser navigation to provide a comparison of the three types of apps. For example, native apps do not provide web browser navigation while web apps allow it. Hybrid apps are simply a combination of the attributes of native apps and web apps. In the second comment, I have deleted the word 'mobile' to avoid confusion.